Documentation/compatibility/uri-reserved-characters-consistent.md

```diff
@@ -0,0 +1,45 @@
+## Ensure System.Uri uses a consistent reserved character set
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In <xref:System.Uri?displayProperty=fullName>, certain percent encoded characters that were sometimes decoded are now consistently left encoded. This occurs across the properties and methods that access the path, query, fragment, or userinfo components of the URI. 
+The behavior will change only when both of the following are true:
+  - The URI contains the encoded form of any of the following reserved characters: " : ", " ' ", " ( ",  " ) ", " ! " or " * ".
```

Comment:
I wonder whether the space between the quotation mark and the symbol will cause confusion (though removing it also causes confusion). Fencing them may help: \`:\`, \`'\`, \`(\`, \`)\`, \`!\`, or \`*\`.

Reply:
I'll do that instead -- this was originally in a document that didn't support markdown and despite spending a lot of time thinking about it I never found a good way to list the symbols.